app.js

```diff
@@ -0,0 +1,68 @@
+// var question = 'what\'s your name?';
```

Comment:
If you don't want to use escape characters, you can do this with double quotes i.e:
`var question = "what's your name?";`